### ISSUE TITLE
Training on other backbone and datasets

### ISSUE THREAD

**Issue by pp00704831** [sadeepj/crfasrnn_pytorch#3]
Hello, I want to ask that if these pytorch crfasrnn model can be used for training?
I want to train a model with other backbone instead of FCN.
By the way, if I want to train on pascal_context dataset, after I change the label number to 59, and reinstall setup.py install, then it might should work smoothly? 
Thanks you !

**Comment by yyyjia** [comment]
Did you succeed?

**Comment by heng-yuwen** [comment]
Hi @yyyjia @pp00704831 , did you succeed training with crf-rnn? Do you know how to train with batch_size larger than 1?

**Comment by alsilence** [comment]
Hi, @pp00704831 @yyyjia  ,did you succeed training with crf-rnn?

**Comment by LuizGuzzo** [comment]
did anyone succeed?
